api/docs/release.dox

```diff
@@ -132,6 +132,9 @@ changes:
  - Added new fields elf_path and elf_path_size to dr_memory_dump_spec_t. When
```

Comment:
style: Prefix the title with sthg like `i#7287 cache policies: ` -- maybe `i#7287 cache policies: Add a release note on the refactoring`.